backend/src/main/java/hu/flowacademy/konyhatunder/repository/RecipeRepository.java

```diff
@@ -4,6 +4,9 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface RecipeRepository extends JpaRepository<Recipe,String> {
+    List<Recipe> findByIngredientsIngredientId(String id);
```

Comment:
Ez kicsit zavaros hogy mit is kene csinaljon es hogy. Valami ilyesmi kene: findByIngridients, vagy findByIngidientId.  (Ha a jpa nem eszi meg lehet jpql queryt is irni.)

Reply:
Külön issueba kiraktam ezt, hogy tudjunk mergelni.

Reply:
Done.